src/main/java/org/folio/rest/impl/CheckinReceivePiecesHelper.java

```diff
@@ -63,13 +47,16 @@
 public abstract class CheckinReceivePiecesHelper<T> extends AbstractHelper {
 
   private static final String PIECES_WITH_QUERY_ENDPOINT = resourcesPath(PIECES) + "?limit=%d&lang=%s&query=%s";
+  private static final String GET_PURCHASE_ORDERS = resourcesPath(PURCHASE_ORDER) + SEARCH_PARAMS;
```

Comment:
`PurchaseOrderHelper.GET_PURCHASE_ORDERS` can be public to avoid the same constants in different places.